.gitignore

```diff
@@ -40,3 +40,4 @@ yarn-debug.log*
 .yarn-integrity
 .vagrant/
 coverage/
+.env
```

Comment:
Actually, I'd keep it and instead ignore `.env.local`

This way, we have a common set of env-vars in `.env` and anyone can add theirs at `.env.local` which we'd be likely to ignore

Reply:
Oh, good idea